conf/help.txt

```diff
@@ -116,9 +116,13 @@ jobchange: "Params: <job name|ID>\n" "Changes your job.\n"
 	"4100 Baby Mechanic       4101 Baby Glt. Cross  4102 Baby Royal Guard  4103 Baby Sorcerer\n"
 	"4104 Baby Minstrel          4105 Baby Wanderer   4106 Baby Sura             4107 Baby Genetic\n"
 	"4108 Baby Shadow Chaser\n"
+	"---- Expanded Baby Class ----\n"
+	"4220 Baby Summoner        4222 Baby Ninja        4223 Baby Kagero         4224 Baby Oboro\n"
+	"4225 Baby Taekwon       4226 Baby Star Glad    4227 Baby Soul Linker    4228 Baby Gunslinger\n"
+	"4229 Baby Rebellion\n"
 	"---- Modes And Others ----\n"
 	" 22 Wedding            26 Christmas          27 Summer           28 Hanbok\n"
-	" 4048 Star Gladiator (Union)\n"
+	" 4048 Star Gladiator (Union)	    4238 Baby Star Glad (Union)\n"
```

Comment:
i guess u r seeing 4038 for the baby star gladiator. well, in case of 4038, it is should be in front of the adult version.